### DISCUSSION TITLE
Release Steps on Github Actions fail to run

### DISCUSSION THREAD

**Discussion by hhimanshu** [semantic-release/semantic-release#1877] [General] [answered]
## Current behavior

Hello 
I am trying semantic-release for the first time. I used the [GitHub Actions](https://github.com/semantic-release/semantic-release/blob/1405b94296059c0c6878fb8b626e2c5da9317632/docs/recipes/github-actions.md) configurations as posted, and currently, the release job fails with the following error

```sh
Run npm ci
npm ERR! cipm can only install packages with an existing package-lock.json or npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or later to generate it, then try again.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-04-08T01_11_40_500Z-debug.log
Error: Process completed with exit code 1.
```
The job is [available on GitHub](https://github.com/hhimanshu/ts-react-components-lib/runs/2292708968?check_suite_focus=true)

The [package.json](https://github.com/hhimanshu/ts-react-components-lib/blob/master/package.json) was created using `npx semantic-release-cli setup` command as described on the [Getting Started page](https://semantic-release.gitbook.io/semantic-release/usage/getting-started)

Please let me know if I am making any mistakes and how to fix this. Thank you

## Expected behavior

The release process should have worked

## Environment

- **semantic-release** version: ^17.4.2
- CI environment: Github Action
- Plugins used: All default ones, no overridden configuration
- **semantic-release** configuration: https://github.com/hhimanshu/ts-react-components-lib/blob/master/package.json#L88-L90
- CI logs: https://github.com/hhimanshu/ts-react-components-lib/runs/2292708968?check_suite_focus=true

**Comment by travi** [comment, marked as answer]
your build is failing before it even gets to the release step. semantic-release is not your problem.

based on the error, you did not commit your lockfile, but are attempting to install with `npm ci`, which requires a lockfile to be present. either commit the lockfile that is generated locally when running `npm install` or use `npm install` to install your dependencies in your workflow rather than `npm ci`
